### ISSUE TITLE
regression in 2.7.0: false positive no-member in numpy

### ISSUE THREAD

**Issue by sam-s** [PyCQA/pylint#4134]
### Steps to reproduce

```python
import numpy as np
print(np.finfo(np.float64).resolution)
```

### Current behavior

```
[E1101(no-member), ProbEval.__init__] Instance of 'finfo' has no 'resolution' member
```

### Expected behavior

no diagnostics, like it was in 2.6.0

### pylint --version output

```
pylint 2.7.0
astroid 2.5
Python 3.9.2 (default, Feb 19 2021, 17:09:53) 
[Clang 12.0.0 (clang-1200.0.32.29)]
```

**Comment by hippo91** [comment]
@sam-s thanks for the report. With current `astroid`'s master branch the problem no longer exists.
Bisecting the `astroid`'s commit tree, i found that PyCQA/astroid@5d547b2670eeead968da40494b9d0ce33237cc61 seems to be the commit that fixed this issue.
